### PR TITLE
Pr62

### PR DESCRIPTION
**Pull request by wraithgar**
Updated PR https://github.com/AmpersandJS/ampersand-select-view/pull/62 with test added, current with master.

Will need to get tests passing before we can merge.

### CODE REVIEW

File: ampersand-select-view.js [hunk 1]
```diff
@@ -47,9 +47,7 @@ module.exports = View.extend({
         }
 
         this.el = opts.el;
-        if (opts.label === undefined) {
-            this.label = this.name;
-        } else {
+        if (opts.label !== undefined) {
```
Comment (wraithgar) [inline ampersand-select-view.js]: This `if` could probably be removed since we check for `this.label` in the render function now.

Comment (kamilogorek) [inline ampersand-select-view.js]: We can set it as `this.label = opts.label || '';` and get rid of `render` check. It should work the same.

Comment (wraithgar) [inline ampersand-select-view.js]: Good idea, I think that will work just fine.

**Comment by wraithgar** [comment]
The original PR had two +1's and this is just getting testing to work, merging.

**Comment by wraithgar** [comment]
@cdaringe this is still a major right?  There are no other pending PRs so I think we can release this whenever you want.

**Comment by cdaringe** [comment]
Ya definitely so.  I can publish in an hour or so.  I'll make sure to patch the changelog too

**Comment by cdaringe** [comment]
published `6.0.0`.  thanks @wraithgar!  you've had a very busy, very ampersandy week :)

**Pull request merged**